docroot/modules/custom/yukon_migrate/migrations/paragraphs/yukon_migrate_secondary_content.yml

```diff
@@ -0,0 +1,34 @@
+id: yukon_migrate_secondary_content
+label: Yukon Migrate Secondary Content field Paragraph
+migration_tags:
+  - Drupal 7
+  - Content
+  - Paragraphs Content
+migration_group: legacy
+source:
+  plugin: d7_paragraphs_item
+  bundle: secondary_content
+  constants:
+    default_text_format: full_html
+process:
+
+    field_category_title: field_category_title
+    field_subcategory_links:
+        -   plugin: sub_process
+            source: field_subcategory_links
```

Comment:
According to data from the database, this field contains links like 'node/$nid'. During the migration process, we don't keep node ids, therefore we need to replace old node URLs with the new correct ones. For this purpose, we have a new process plugin which is called - Links. Could you please re-work this section to make it use this plugin?

Reply:
This is now resolved.